### PR TITLE
Clarified README.md

### PR DESCRIPTION
**Pull request by tomschlenkhoff**
Added experiences from me updating my Thinkpad.

### CODE REVIEW

File: x230/README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Get it from our [release page](https://github.com/merge/coreboot-x230/releases)
 * __SeaBIOS__: version [1.12.0](https://seabios.org/Releases) from 2018-11-17
 
 
-## table of contents
+## Table of contents
```
Comment (merge) [inline x230/README.md]: unrelated typo fix, if at all

File: x230/README.md [hunk 2]
```diff
@@ -18,32 +18,29 @@ Get it from our [release page](https://github.com/merge/coreboot-x230/releases)
...
+3. Connect a hardware flasher to an external PC (or a Raspberry Pi with a SPI 8-pin chip clip
+can directly be used)
+4. Run `sudo ./external_install_bottom.sh` on the lower chip
+5. Run `sudo ./external_install_top.sh` on the top chip of the two
+6. Optionally: For updating later, run `./x230_skulls.sh`. No need to disassemble.
 
 And always use the latest [released](https://github.com/merge/coreboot-x230/releases)
-package. This will be tested. The git master
-branch is _not_ meant to be stable. Use it for testing only.
+package. This will be tested. The git master branch is _not_ meant to be stable. Use it for testing only.
```
Comment (merge) [inline x230/README.md]: this is all random refactoring. and _if_ you want that should be a seperate commit

File: x230/README.md [hunk 3]
```diff
@@ -18,32 +18,29 @@ Get it from our [release page](https://github.com/merge/coreboot-x230/releases)
...
+
 Before flashing coreboot, consider doing one original Lenovo upgrade process
 in case you're not running the latest version. This is not supported anymore,
 once you're running coreboot (You'd have to manually flash back your backup
 images first, see later chapters).
 
-Also, this updates the BIOS _and_ Embedded Controller (EC) firmware. The EC
+Check the [Lenovo Support site (which is quite good and actually helpful)](https://pcsupport.lenovo.com/de/en/products/laptops-and-netbooks/thinkpad-x-series-laptops/thinkpad-x230) and e.g. run the _Lenovo System Update for Windows_ to check for old BIOS, EC- or battery-firmware or other updateable firmwar.
+
+Also, this updates the BIOS (latest 2.74) _and_ Embedded Controller (EC) firmware. The EC
```
Comment (merge) [inline x230/README.md]: no, sorry. first, we don't differentiate between running windows or linux here. Lenovo provides bootable ISO. Also, we don't want to maintain the latest BIOS version number in the README too.

File: x230/README.md [hunk 4]
```diff
@@ -126,14 +132,14 @@ or ethernet to `sudo apt-get install flashrom`
...
+		 L           GND                                  CS
+		 E            |                                   |
 		 F +---------------------------------------------------------------------------------+
 		 T |  x   x   x   x   x   x   x   x   x   x   x   x   x   x   x   x   x   x   x   x  |
 		   |  x   x   x   x   x   x   x   x   x   x   x   x   x   x   x   x   x   x   x   x  |
 		 E +----------------------------------^---^---^---^-------------------------------^--+
-		 D                                    |   |   |   |                               |
-		 G                                   3.3V MOSIMISO|                              GND
+		 D                                    |   |   |   |                              
+		 G                                   3.3V MOSIMISO|
```
Comment (merge) [inline x230/README.md]: why would you remove the uart pins?

File: x230/README.md [hunk 5]
```diff
@@ -161,57 +167,62 @@ based supply for a second USB port (like [this](https://de.aliexpress.com/item/1
...
-	sudo ./external_install_bottom.sh -m -k <backup-file-to-create>
+	sha256sum skulls-x230-0.1.0.tar.xz anc compare this to the [checksum](https://github.com/merge/skulls/releases/download/0.1.0/skulls-x230-0.1.0.tar.xz.sha256)
 
+### Side note
+Flashing with these low speeds takes time. Be patient. E.g. unlocking the bottom chip with its two reads, one write and one verify step usually takes one hour in total. Again, be patient!
 
-That's it. Keep the backup safe.
+## First, optional step: Connect to the bottom chip
+There are a few reasons why you may start with connecting your clip to the bottom (at the bottom, closer to you) chip (it has the same pinout than the upper chip):
+- You may want to enable in system updates in the future. The advantage is that you can update and change whatever you decide to flash in the upper chip. The disadvantage is that any software can flash you BIOS with this setting. Choose wisely (Heads - see below - may be of use here).
```
Comment (merge) [inline x230/README.md]: that might be useful actually. documenting this chip to be optional might be a good idea...

File: x230/README.md [hunk 6]
```diff
@@ -187,7 +181,9 @@ Make sure to verify the checksum with e.g.:
 	sha256sum skulls-x230-0.1.0.tar.xz anc compare this to the [checksum](https://github.com/merge/skulls/releases/download/0.1.0/skulls-x230-0.1.0.tar.xz.sha256)
 
 ### Side note
-Flashing with these low speeds takes time. Be patient. E.g. unlocking the bottom chip with its two reads, one write and one verify step usually takes one hour in total. Again, be patient!
+Flashing with these low speeds takes time. Be patient. E.g. unlocking the bottom chip with its two reads, one write and one verify step usually takes approximately one hour in total. Again, be patient!
```
Comment (merge) [inline x230/README.md]: might be worth adding if you make one single patch with this addition...

File: x230/README.md (hunk 1 above)
Comment (tomschlenkhoff) [reply x230/README.md]: True, just wanted to unify headings. Not important.

File: x230/README.md (hunk 2 above)
Comment (tomschlenkhoff) [reply x230/README.md]: True, just happened to fix it in passing. Not important.

File: x230/README.md (hunk 3 above)
Comment (tomschlenkhoff) [reply x230/README.md]: It would make sense to differentiate between Linux/Windows because coming from Windows it is not possible to run "x230_before_first_install.sh" without further preparation.
Just added the BIOS version because EC version was in the text. Not important.

File: x230/README.md (hunk 4 above)
Comment (tomschlenkhoff) [reply x230/README.md]: Because the are not used and create confusion at first. Just kept the connected pins.

File: x230/README.md [hunk 7]
```diff
@@ -263,3 +266,7 @@ Mainly a tiny binary used by the Ethernet card and the Intel Management Engine.
 * `git clone https://github.com/merge/skulls`
 * rename one of the included config files to `config-xxxxxxxxxx`.
 * The x230 directory's `./build.sh` should produce the exact corresponding release image file.
+
+## Further reading / more pictures
+* [Flashing the X230 with coreboot while slaying Intel-ME](https://steemit.com/tutorial/@joeyd/run-don-t-walk-from-the-blob)
+* [Step by Step easy guide to flashing Coreboot (X230, but should work for others too)](https://www.reddit.com/r/thinkpad/comments/4zrmf8/step_by_step_easy_guide_to_flashing_coreboot_x230/)
```
Comment (merge) [inline x230/README.md]: let's avoid linking to tutorials and add everything we need to the README.

Comment (tomschlenkhoff) [reply x230/README.md]: I found the pictures in the steemit-tutorial. But not strictly required.

**Comment by tomschlenkhoff** [comment]
Which changes do you want me to move to one bespoke commit?

**Comment by merge** [comment]
well. for one interesting addition (I've found 2 that I find interesting, see above), create one commit. and if possible write commit messages like "warn users about long waiting times" (for example) rather than "further readings".
